.github/workflows/release-report.yml

```diff
@@ -0,0 +1,42 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Checks that are intended to publish PRs test results from JUnit.
+
+name: Java PR Report
+on:
+  workflow_run:
+    workflows:
+      - Release
+    types: [ completed ]
```

Comment:
Is there a reason to have this as a separate workflow? Can we just have this as a step (or separate) at the end of the release?

Its not clear to me what we're trying to accomplish here, but it seems like this would be more valuable as a status on the release run itself rather than a different workflow

Reply:
If we want to reuse these steps across multiple workflows (which seems like the right thing to do), we should make this a composite action - https://docs.github.com/en/actions/creating-actions/creating-a-composite-action - which can be called by both the pr and release workflows. I don't think we want a separate workflow to do this at all.

In practice, this is just 2 steps so I wouldn't even bother, I'd just add these actions to both workflows directly